src/injector/stage1/src/main.rs

```diff
@@ -88,11 +82,193 @@ fn main() {
         .unpack("/zarf-stage2")
         .expect("Unable to unarchive the resulting tarball");
 
-    // make all stage2 files executable
-    for entry in glob("/zarf-stage2/**/*").unwrap() {
-        match entry {
-            Ok(path) => chmod755(path.to_str().unwrap()),
-            Err(e) => println!("{:?}", e),
+    let mut seed_image_tar = Archive::new(File::open("/zarf-stage2/seed-image.tar").unwrap());
+    seed_image_tar
+        .unpack("/zarf-stage2/seed-image")
+        .expect("Unable to unarchive the seed image tarball");
+}
+
+/// Starts a static docker compliant registry server that only serves the single image from /zarf-stage2/seed-image
+///
+/// (which is a docker registry image in crane's tarball format) - https://github.com/google/go-containerregistry/tree/main/pkg/v1/tarball
```

Comment:
In order to fully support image digests in the future we may need to move to an OCI storage format vs tarball.  We could keep the injector as a legacy format, but curious if you have thoughts based on your work here with OCI vs docker save tarball format.

Reply:
OCI storage format will be even easier, as we can just leverage `oci_spec` crate to `serde` out the manifest. We wouldnt even need the compatibility layer between crane and V2.  (I am doing media type overrides on the OCI spec to make it V2 spec)

Reply:
Since we already handle the seed image in a special way, this is the perfect time to introduce OCI save for this function. I think it will be a cleaner overall implementation that way.

Reply:
Agree on that being much cleaner.